src/test/infrastructure/nfs-server/values.yaml

```diff
@@ -56,7 +56,10 @@ service:
 persistence:
   annotations: {}
   size: 5Gi
-  storageClassName: "gp2"
+  # If set to nonempty string value, this will specify the storage class to be used.
+  # If left without value, the default Storage Class will be utilised.
```

Comment:
Can we stipulate what the default storage class is i.e. `gp2`

Reply:
This probably depends on the cluster, I am not sure if we assume a default class here.